src/edu/stanford/dlss/was/WasapiDownloaderSettings.java

```diff
@@ -25,6 +25,7 @@
   public static final String JOB_ID_PARAM_NAME = "jobId";
   public static final String CRAWL_START_AFTER_PARAM_NAME = "crawlStartAfter";
   public static final String CRAWL_START_BEFORE_PARAM_NAME = "crawlStartBefore";
+  public static final String OUTPUT_DIR_PARAM_NAME = "outputDir";
```

Comment:
this is fine;  just wondering if we want to call out that it's more of an outputBASEdir.  Don't have strong feelings about changing it.